py-polars/polars/expr/string.py

```diff
@@ -103,6 +107,10 @@ def to_datetime(
         exact
             Require an exact format match. If False, allow the format to match anywhere
             in the target string.
+
+            .. note::
+                This introduces a performance penalty - cleaning your data beforehand
+                will almost certainly be more performant.
```

Comment:
It's not immediately clear which setting incurs the performance penalty - True or False?

Reply:
@MarcoGorelli You missed the docstrings in `series.py`!

Reply:
very good point, I should've thought of that - thanks!